package.json

```diff
@@ -20,12 +21,13 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "^5.0.0",
```

Comment:
Everything works when you run `npm start`, but there's an ugly warning.

```
WARNING in DefinePlugin
Conflicting values for 'process.env'
```

Reply:
@jaredperreault-okta Do you have any idea how to fix this?

Reply:
Not sure what the issue is, but the problem is here: https://github.com/okta-samples/okta-react-sample/blob/main/config-overrides.js#L43.

`react-scripts` contains a webpack config internally, `config-overrides` allows us to customize it. It seems like our customization is colliding with something in `react-scripts` 5

Reply:
I can't seem to get the warning to go away. I think I'll just leave it and hope that someone has a solution someday (if it annoys them). `npm start` and `npm run build` work just fine.

Reply:
Damn. It doesn't work for CI because `Treating warnings as errors because process.env.CI = true.`

https://github.com/okta-samples/okta-react-sample/runs/6080989731?check_suite_focus=true